server/router.js

```diff
@@ -0,0 +1,25 @@
+// import express, express.router, & controller file
+const express = require('express');
+const router = express.Router();
+const controller = require('./controller');
```

Comment:
`controller` is a bit generic - it might make sense to call this 'getAllData' or something along those lines, since that's the function you're importing/exporting for use

Reply:
`controller` sounds good to me if you're going to be adding other methods to that file